torchreid/integration/nncf/compression.py

```diff
@@ -229,3 +229,17 @@ def wrap_inputs(args, kwargs):
                                                       compression_state=resuming_state_dict)
 
     return compression_ctrl, model, nncf_metainfo
+
+def get_nncf_complession_stage():
+    try:
```

Comment:
I encountered a problem on the cluster with nncf has not been installed. So, your PR is useful. But if we have nncf as a dependency, maybe we add it to optional requirements?